### PR TITLE
Prepare beta based on ee master

### PR DESCRIPTION
**Pull request by DominikDary**
@fatz please review

### CODE REVIEW

File: aws/windows-agent/main.tf [hunk 1]
```diff
@@ -29,12 +29,26 @@ module "dcos" {
   bootstrap_instance_type = "m4.xlarge"
 
   dcos_variant              = "ee"
-  dcos_version              = "1.13.1"
+  dcos_version              = "2.1.0"
   dcos_license_key_contents = "${file("~/license.txt")}"
+  ansible_bundled_container = "mesosphere/dcos-ansible-bundle:windows-beta-support"
+
+  #Linux Installer path - place url with "pull/PR#" or "master" suffix here:
+  custom_dcos_download_path = "https://downloads.mesosphere.com/dcos-enterprise/testing/master/dcos_generate_config.ee.sh"
```
Comment (fatz) [inline aws/windows-agent/main.tf:37]: I see the point here but for a tested version of this example with a stable result an immutable release should be chosen. E.g. a commit-hash which was verified to work with this example. 

If this stay with master results will be different every time this example is ran by someone

File: aws/windows-agent/main.tf [hunk 2]
```diff
@@ -29,12 +29,26 @@ module "dcos" {
...
   dcos_superuser_username      = "demo-super"
+
+  dcos_config = <<-EOF
+enable_windows_agents: true
+-EOF
+
+  #Windows Installer path - place url with "pull/PR#" or "master" suffix here:
+ansible_additional_config = <<-EOF
+dcos:
+ download_win: "https://downloads.mesosphere.com/dcos-enterprise/testing/master/windows/dcos_generate_config_win.ee.sh"
```
Comment (fatz) [inline aws/windows-agent/main.tf:50]: I see the point here but for a tested version of this example with a stable result an immutable release should be chosen. E.g. a commit-hash which was verified to work with this example. 

If this stay with master results will be different every time this example is ran by someone

**Review by fatz — CHANGES_REQUESTED** [review]
For this being a reproducible and tested example we should use immutable download URLs e.g. with a given commit hash

File: aws/windows-agent/main.tf (hunk 1 above)
Comment (DominikDary) [reply aws/windows-agent/main.tf:37]: for now master artifact is good enough.

File: aws/windows-agent/main.tf (hunk 2 above)
Comment (DominikDary) [reply aws/windows-agent/main.tf:50]: for now master artifact is good enough.

**Pull request merged**